src/pages/Chat/Chat/index.tsx

```diff
@@ -99,8 +162,31 @@ export default (): JSX.Element => {
   // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  // Initial load messages when chatRef is set
   useEffect(() => loadMessages(), [loadMessages, chatRef]);
 
+  // Get real time chat updates
+  useEffect(() => {
+    if (!chatRef) return () => null;
+
+    firstLoad.current = true;
+    const unsubscribe = chatRef
+      .onSnapshot((chatSnapshot) => {
+        const chatSnapshotData = chatSnapshot.data();
+        if (chatSnapshotData?.currentlyTyping) {
+          const otherIsTypingSnapshot = (chatSnapshotData.currentlyTyping as DocumentRefData[])
+            .map((c) => c.id)
```

Comment:
Minor sugestion: Instead of using a map, filter and `length > 0`, you could use something along the lines of `.some((docRef) => docRef.id !== currentUserRef.id)`.

Reply:
Fixed this.